crates/hyperqueue/src/server/bootstrap.rs

```diff
@@ -90,19 +92,27 @@ pub async fn get_client_connection(server_directory: &Path) -> anyhow::Result<Cl
             )
         })?;
 
-    Ok(connection)
+    Ok(session)
 }
 
 async fn get_server_status(server_directory: &Path) -> crate::Result<ServerStatus> {
     let record = ServerDir::open(server_directory).and_then(|sd| sd.read_access_record())?;
 
-    if HqConnection::connect_to_server(&record).await.is_err() {
+    if ClientSession::connect_to_server(&record).await.is_err() {
         return Ok(ServerStatus::Offline(record));
     }
 
     Ok(ServerStatus::Online(record))
 }
 
+fn generate_server_uid() -> String {
```

Comment:
We could also do something like `Docker` and generate short names like `religious-squirrel` or `bountiful-bear` or something like that :)

Reply:
I want to use it in paths and I want to have rather shorter paths:)